R/modify-model-field.R

```diff
@@ -236,6 +236,19 @@ replace_description <- function(.mod, .description) {
   )
 }
 
+#' @describeIn modify_model_field Fills the description field in a model object and corresponding YAML, if it is currently empty.
+#' @importFrom checkmate assert_scalar
+#' @export
+add_description <- function(.mod, .description) {
```

Comment:
Honest opinion: this seems complicated having `add_` and `replace_`.   Can we just do `set_description()` or whatever  and leave it at that?

Reply:
This was kind of a legacy thing because we already had `replace_description` when it was a required field.

The reason both exist now is because 1) we already had `replace_` and I didn't want to deprecate it and 2) it seemed like (if I didn't make `add_`) people would instinctively want to type `add_description()` and be confused as to why they had to use `replace_` when they weren't actually replacing anything.